packages/components-library/src/components/filters/CheckboxFilter.vue

```diff
@@ -60,6 +70,12 @@ export default {
       type: Array,
       default: () => []
     },
+
```

Comment:
Please add some documentation here, like the other props have

Reply:
Fixed in 1c81229